modules/subscriptions.php

```diff
@@ -670,7 +670,7 @@ function comment_subscribe_init( $submit_button, $args ) {
 		 */
 		$str = apply_filters( 'jetpack_comment_subscription_form', $str );
 
-		return $str . $submit_button;
+		return $submit_button . $str;
```

Comment:
@jeherve I think this flipped things to put the submit button first.

See https://wordpress.org/support/topic/comment-form-tickboxes-showing-after-submit-comment-button/

Reply:
ugh, weren't they after the submit button before? I remembered wrong. I'll get this fixed.

Reply:
#12373. My bad!

Reply:
I looked at it and swear I didn't catch they moved. On me too.